noxfile.py

```diff
@@ -13,6 +13,9 @@
 
 @nox.session
 def format(session):
+    """
+    Formats the files to remove errors
```

Comment:
please add periods at the end of docstrings.

Here I would also just write `Autoformat source files.`